app/[lang]/chapters/[slug]/layout.tsx

```diff
@@ -1,13 +1,15 @@
+'use client'
```

Comment:
Any particular reason why this needed to become a Client component?

Reply:
I had used it earlier when I was calling hooks in this file. But after refactoring the code, I had forgotten to remove this.

Thanks for catching this.

Addressed in 9b4c5743cb1375d2eae011ab956848b2f92682c6